content/millennium/dstu2/individuals/patient.md

```diff
@@ -145,6 +147,88 @@ The ability to perform patient combine or uncombine operations is not available
 
 The common [errors] may be returned.
 
+## Create
+
+Create an individual Patient:
+
+    POST /Patient
+
+_Implementation Notes_
+
+* The following elements are not supported and will not be used while creating the patient:
```

Comment:
Maybe clarify to "and will cause an error to be returned if set" instead of stating they'll not be used. It makes it sound like we'll ignore them :)

Reply:
updated the note to 'will cause an error to be returned if set'